src/main/java/io/vertx/spi/cluster/zookeeper/ZookeeperClusterManager.java

```diff
@@ -37,7 +41,7 @@
  *
  * @author Stream.Liu
  */
-public class ZookeeperClusterManager implements ClusterManager, PathChildrenCacheListener, ConnectionStateListener {
+public class ZookeeperClusterManager implements ExtendedClusterManager, PathChildrenCacheListener, ConnectionStateListener {
```

Comment:
en, it true, i should remove **beforeLeave** method after look into the Hazelcast Cluster.
zookeeper will remove these lock nodes if connection lost.